ui/src/app/console/Console.tsx

```diff
@@ -133,6 +137,34 @@ export default function Console() {
       });
   };
 
+  const handleCopyAsCli = (values: ConsoleFormValues) => {
+    let parsed = null;
+    try {
+      // need to unescape the context string
+      parsed = JSON.parse(values.context);
+    } catch (err) {
+      setHasEvaluationError(true);
+      setError('Context provided is invalid.');
+      return;
+    }
+    const command = generateCliCommand({
+      commandName: Command.Evaluate,
+      arguments: [values.flagKey],
+      options: [
+        { key: '--entity-id', value: values.entityId },
+        { key: '--namespace', value: namespace.key },
+        {
+          key: '--context',
+          value: Object.entries(parsed)
+            .map(([key, value]) => `${key}=${value}`)
+            .join(' ')
+        }
+      ]
+    });
```

Comment:
Thank you for your initial contribution @kyryl-perepelytsia.

There is a small adjustment needed for the CLI command regarding the context option. If there are multiple context values, each one should be passed as a separate option, such as `--context apple=a --context pear=p`.

```suggestion
    const contextOptions = Object.entries(parsed).map(([key, value]) => {
      return {
        key: '--context',
        value: `${key}=${value}`
      };
    });
    const command = generateCliCommand({
      commandName: Command.Evaluate,
      arguments: [values.flagKey],
      options: [
        { key: '--entity-id', value: values.entityId },
        { key: '--namespace', value: namespace.key },
        ...contextOptions
      ]
    });
```

Reply:
Thanks, updated